src/layouts/MainSection.js

```diff
@@ -1,22 +1,43 @@
-import AnswerSection from '../components/AnswersSection/AnswersSection';
 import NewGameButton from '../components/NewGameButton';
-import MainImage from '../components/MainImage';
+import RankSection from '../components/RankSection/RankSection';
+import RulesSection from '../components/RulesSection/RulesSection';
+import rules from '../model/rules';
 
 const MainSection = () => {
-  const mainSection = document.createElement('div');
+  const mainSection = document.createElement('section');
   mainSection.classList.add('mainSection');
 
-  const answers = ['Walter White', 'Jesse Pinkman', 'Hank Shrader', 'Saul Goodman'];
-  const handler = (isCorrect, answer) => {
-    console.log(answer, isCorrect);
-  };
+  const menuSection = document.createElement('section');
+  menuSection.classList.add('menuSection');
+  const contentSection = document.createElement('div');
+  contentSection.classList.add('contentSection');
 
-  mainSection.append(AnswerSection({ answers, correctAnswer: 'Walter White' }, handler));
   const startGame = () => {
     console.log('The game has started');
   };
-  mainSection.append(NewGameButton(startGame));
-  mainSection.appendChild(MainImage());
+
+  const rulesRankChange = (e) => {
+    console.log(e);
+    if (e.target.textContent === 'Rules')
+      e.target.innerHTML =
+        '<span class="newGameButton__firstLetter">H</span><span class="newGameButton__restOfText">all Of Fame</span>';
+    else if (e.target.textContent === 'Hall Of Fame')
```

Comment:
Wyjaśnij proszę kiedy to wykorzystujesz :)

Reply:
To tak na chwilę testowo sobie zrobiłem, żeby zmieniąć nazwę przycisku z Rules na HallOfFame :P Już tego nie ma :)